packages/backend/src/graphql/schema.graphql

```diff
@@ -33,8 +33,11 @@ type Query {
     sortOrder: SortOrder
   ): PeersConnection!
 
-  "Fetch a wallet address"
-  walletAddress(id: String!): WalletAddress
+  "Fetch a wallet address, and optionally additional properties."
+  walletAddress(
+    id: String!
+    includeAdditionalProperties: Boolean
+  ): WalletAddress
```

Comment:
I think controlling the return of `additionalProperties` with `includeAdditionalProperties` here is a kind of gql anti-pattern.

Consider this is how we query for it:

```ts
query WalletAddress($walletAddressId: String!) {
  walletAddress(
    id: $walletAddressId
    includeAdditionalProperties: true
  ) {
    id
    liquidity
    asset {
      code
      scale
    }
    url
    publicName
    status
    additionalProperties {
      properties {
        key
        value
        visibleInOpenPayments
      }
    }
  }
}
```

We are already specifying that we want `additionalProperties` by the query itself. That should drive whether or not we fetch/return them - the argument is superfluous and complicating (what happens if we specify the field in the query but `includeAdditionalProperties` is `false`? and vice-versa). 

Instead of including this argument, we can make a separate resolver for `additionalProperties`, so the `rafiki/packages/backend/src/graphql/resolvers/index.ts` would look like:

```ts
  WalletAddress: {
    liquidity: getWalletAddressLiquidity,
    incomingPayments: getWalletAddressIncomingPayments,
    outgoingPayments: getWalletAddressOutgoingPayments,
    quotes: getWalletAddressQuotes,
    walletAddressKeys: getWalletAddressKeys,
    additionalProperties: getAdditionalProperties
  },
```

This way, every time the Wallet Address resolves, it will then resolve the additionalProperties if it was specified in the query. Like it does for `liquidity`, `incomingPayments`, etc.